source/Preferences.cpp

```diff
@@ -204,43 +205,29 @@ void Preferences::SetScrollSpeed(int speed)
 // View zoom.
 double Preferences::ViewZoom()
 {
-	return ZOOMS[zoomIndex];
+	return zoom;
 }
 
 
 
 bool Preferences::ZoomViewIn()
 {
-	if(zoomIndex == static_cast<int>(ZOOMS.size() - 1))
-		return false;
+	const Interface *hudInterface = GameData::Interfaces().Get("hud");
```

Comment:
instead of requesting this every time over and over again, lets just load this once and store it in a variable as interfaces do not change during gameplay.
you can decide if you want to store the interface or the individual members of it.

Reply:
not really! interface values will be customizable in the future, so we cant assume it does not change
+ this is how it's done everywhere right now

Reply:
Agreed that we should just rerequest the value. This isn't performance intensive code, and as Hurl says, things could change in the future where the value is not a constant.

Reply:
hmm, ok